test/test_oauth.py

```diff
@@ -101,7 +101,11 @@ def _oauth_request(self, endpoint, params=None,
             oauth_token, oauth_token_secret,
         )
         try:
-            return session.get(TESTSERVER + endpoint, params=params).text
+            connection = session.get(TESTSERVER + endpoint, params=params)
+            if connection.status_code == 200:
+                return connection.text
+            else:
+                raise unittest.SkipTest()
```

Comment:
```suggestion
            connection.raise_for_status()
            return connection.text
```
We should just call `raise_for_status()` here to avoid code duplication.

It might also be sensible to `raise unittest.SkipTest()` unconditionally at the beginning of `_oauth_request()` since the public test server at `term.ie` is not available anymore and hasn't been for some time.